apps/avatax/src/modules/avatax/order-cancelled/avatax-order-cancelled-payload-transformer.ts

```diff
@@ -5,12 +5,8 @@ import { AvataxOrderCancelledTarget } from "./avatax-order-cancelled-adapter";
 export class AvataxOrderCancelledPayloadTransformer {
   constructor() {}
 
-  transform({ order }: OrderCancelledPayload, companyCode: string): AvataxOrderCancelledTarget {
-    if (!order) {
-      throw new Error("Order is required");
-    }
-
-    const transactionCode = z.string().min(1).parse(order.avataxId);
+  transform(avataxId: string, companyCode: string): AvataxOrderCancelledTarget {
```

Comment:
This syntax is risky, very easy to change arguments order and Typescript will not help.
I suggest object destructuring